### PR TITLE
Added include_in_all option to mapping in literature.

### PR DESCRIPTION
**Pull request by pazembrz**
Added include_in_all option to mapping in literature. Added tests to check if it behaves properly

### CODE REVIEW

**Comment by michamos** [comment]
aren't you including too much here? for example refs to other records will also be included. In the issue, I detailed every subfield that should be included.

File: inspirehep/modules/records/mappings/v5/records/hep.json [hunk 1]
```diff
@@ -1005,6 +1047,7 @@
                     "type": "text"
                 },
                 "thesis_info": {
+                    "include_in_all": true,
```
Comment (michamos) [inline inspirehep/modules/records/mappings/v5/records/hep.json]: you should exclude the linked record here.

File: inspirehep/modules/records/mappings/v5/records/hep.json [hunk 2]
```diff
@@ -1051,15 +1094,18 @@
...
                         },
                         "title": {
+                            "include_in_all": true,
                             "type": "text"
                         }
                     },
                     "type": "object"
                 },
                 "titles": {
+                    "include_in_all": true,
```
Comment (michamos) [inline inspirehep/modules/records/mappings/v5/records/hep.json]: we don't need the source here.

Comment (ammirate) [reply inspirehep/modules/records/mappings/v5/records/hep.json]: good spot :+1:

**Pull request merged**